src/middleware/logger.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * All kinds of arguments can come through
+ *
+ * Examples seen are
+ * - string
+ * - object / hash
+ * - values used for string interpolation, basically anything
+ *
+ * See https://linear.app/inngest/issue/INN-1342/flush-logs-on-function-exitreturns for more details
+ */
+type LogArg = object;
```

Comment:
Would `unknown` work? Every time an engineer writes an `any` an angel gets its wings clipped

Reply:
@goodoldneon 💯

Reply:
I'd prefer to not use `any` either.
I end up with `object` because I got a recommendation with maybe the linter when running `yarn dev`.
I can also just do `string | number | object` too and that should cover pretty much everything.

Reply:
We don't care what type the arguments are, right? I'd just use `unknown`. The `unknown` type is basically the same as `any` but it forces you to do type assertions before doing anything with it.

TypeScript's `unknown` is a lot like Go's `any` (a.k.a. `interface {}`)